Blog overview redesign

Arth101 opened the pull request.

maceq687 committed:
blog and recent posts layout revamp

maceq687 committed:
Fixed display only 3 recent posts

maceq687 committed:
Added smaller version of images for posts thumbnails

maceq687 committed:
Limit recent posts size on big screens

maceq687 committed:
Add gradient overlay to recent posts thumbnails to further de-emphasize them since they are for decoration only

maceq687 committed:
Alligned blog title with posts

maceq687 committed:
Changed latest posts layout to two columns, adjusted the thumbnail gradient opacity and increased padding bellow the section

Arth101 committed:
Redesign blog overview

maceq687 reviewed (approved):
Looking nice and clean :+1:

Arth101 committed:
Overhaul the blog overview page:

- Show full author info.
- Show blogpost language.
- Use layout from newer pages on blog langing (hero, container)
